### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by mikesmeltzer**
Initial creation of Events section.

### CODE REVIEW

**Comment by david-poindexter** [comment]
@mikesmeltzer we are going to put this one on hold for a just a bit.  We worked out a table of contents structure for the **Features** section during today's DNN Docs meetup.  I will be submitting a PR soon.  At that point, we can circle back to this one and get it tidied up a bit for merging.  Thanks!

**Comment by mikesmeltzer** [comment]
@david-poindexter Okay! Thanks for the info. Once your PR is in we can touch base and go from there.

**Comment by david-poindexter** [comment]
@mikesmeltzer I submitted #431 yesterday.  As soon as it is merged I can work with you on where this should go.  Thanks!

File: content/features/events/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+uid: dnn-features
```
Comment (david-poindexter) [inline content/features/events/index.md]: @mikesmeltzer I suggest `uid: event-system`

Comment (david-poindexter) [reply content/features/events/index.md]: @mikesmeltzer hey - just checking back in on this quick change request.  Let me know if you need to discuss it in more detail before proceeding.  Thanks!

Comment (mikesmeltzer) [reply content/features/events/index.md]: @david-poindexter This makes sense to me! I'll look to fix this up later today. Thanks :)

Comment (david-poindexter) [reply content/features/events/index.md]: @mikesmeltzer any status updates on this?  Thanks.

Comment (mikesmeltzer) [reply content/features/events/index.md]: @david-poindexter I think the changes are all made now. Back to you sir :)

Comment (david-poindexter) [reply content/features/events/index.md]: @mikesmeltzer thank you sir!

**Pull request merged**